### PR TITLE
26 create individual activity UI

### PR DESCRIPTION
**Pull request by Woodhouse15**

### CODE REVIEW

**Comment by Woodhouse15** [comment]
Add activities and their functionality

File: app/src/main/java/com/example/feelgoodinc/activityRecyclerView/ActivityData.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/activityRecyclerView/ActivityData.java:1]: This should probably go in `/Models`, and perhaps be renamed to `Activity`

Also, need some Javadoc explaining what the class does

Lastly, for `estimatedTime`, what kind of time does it need to be seconds? milieseconds?

Also, for `imagePath`, relative to where? Where should images typically be stored?

Probably worth adding comments for those.

File: app/src/main/java/com/example/feelgoodinc/activityRecyclerView/ActivityViewHolder.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/activityRecyclerView/ActivityViewHolder.java:1]: my gut tells me this shouldn't be a separate Java file. Maybe it should be in the same file as the rest of the parts of the recycler view (and the fragment/activity) that the view is in?

Not sure what the official best practice is for `RecyclerView` though.

File: app/src/main/java/com/example/feelgoodinc/activityRecyclerView/ClickListener.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/activityRecyclerView/ClickListener.java:1]: I really do feel like this also doesn't need to be a separate java file just for 12 lines of code

Then again, I'm also not sure what the best practice is for this.

File: app/src/main/java/com/example/feelgoodinc/adapters/ActivityGalleryAdapter.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.feelgoodinc.R;
+import com.example.feelgoodinc.activityRecyclerView.ActivityData;
+import com.example.feelgoodinc.activityRecyclerView.ActivityViewHolder;
+import com.example.feelgoodinc.activityRecyclerView.ClickListener;
+
+import java.util.List;
+
+public class ActivityGalleryAdapter extends RecyclerView.Adapter<ActivityViewHolder> {
```
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/adapters/ActivityGalleryAdapter.java:20]: A quick javadoc description here would be good

File: app/src/main/java/com/example/feelgoodinc/fragments/ActivityFragment.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/fragments/ActivityFragment.java:1]: Put this in a sub-directory with the other fragments you created?

File: app/src/main/java/com/example/feelgoodinc/fragments/BreathingFragment.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/fragments/BreathingFragment.java:1]: Put this in a sub-directory with the other fragments you created?

File: app/src/main/java/com/example/feelgoodinc/fragments/GroundingFragment.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/fragments/GroundingFragment.java:1]: Put this in a sub-directory with the other fragments you created?

File: app/src/main/java/com/example/feelgoodinc/fragments/SleepFragment.java
Comment (nimitz-Bear) [inline app/src/main/java/com/example/feelgoodinc/fragments/SleepFragment.java:1]: Put this in a sub-directory with the other fragments you created?

File: app/src/main/res/drawable/birds.jpg
Comment (nimitz-Bear) [inline app/src/main/res/drawable/birds.jpg:1]: image

File: app/src/main/res/drawable/circle_shape.xml
Comment (nimitz-Bear) [inline app/src/main/res/drawable/circle_shape.xml:1]: I have literally no idea how these work so I'm gonna go with the good old: LGTM

File: app/src/main/res/layout/fragment_mood.xml
Comment (nimitz-Bear) [inline app/src/main/res/layout/fragment_mood.xml:1]: Thanks for changing these :)

File: app/src/main/res/layout/fragment_grounding.xml
Comment (nimitz-Bear) [inline app/src/main/res/layout/fragment_grounding.xml:1]: Put into a `layout/calming_activities` sub-folder?

File: app/src/main/res/layout/fragment_breathing.xml
Comment (nimitz-Bear) [inline app/src/main/res/layout/fragment_breathing.xml:1]: Put into a `layout/calming_activities` sub-folder?

File: app/src/main/res/layout/fragment_activity.xml
Comment (nimitz-Bear) [inline app/src/main/res/layout/fragment_activity.xml:1]: Put into a `layout/calming_activities` sub-folder?

File: app/src/main/res/layout/fragment_sleep.xml
Comment (nimitz-Bear) [inline app/src/main/res/layout/fragment_sleep.xml:1]: Put into a `layout/calming_activities` sub-folder?

File: app/src/main/res/values/strings.xml
Comment (nimitz-Bear) [inline app/src/main/res/values/strings.xml:1]: Thanks 🥇

**Review by nimitz-Bear — CHANGES_REQUESTED** [review]
In general, for the `Fragment`s you create for the calming activities, it could be a good idea to create a sub-directory under `/fragments`, something like `calming_activities` to  ensure that all the related fragments are all kept in the same place.

Also, for the `Fragment`s' XML files, `grounding_fragment.xml`, `breathing_fragment.xml` and `sleep_fragment.xml`, it could be good to have a sub-directory in the `/layouts` folder for them.

Overall, quite good, especially for an issue this big. 

It's just a matter of some minor javadocs missing, and moving some of the fragment `.java` and `.xml` files together, so they're more maintainable.

**Review by nimitz-Bear — APPROVED** [review]
LGTM

**Pull request merged**